src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java

```diff
@@ -29,7 +29,10 @@ public LeaseClientProvider getLeaseClientProvider() {
 
     @Bean("storage-client")
     public BlobServiceClient getStorageClient(StorageSharedKeyCredential credentials) {
-        return new BlobServiceClientBuilder().credential(credentials).buildClient();
+        return new BlobServiceClientBuilder()
+            .credential(credentials)
+            .endpoint(String.format("https://%s.blob.core.windows.net", credentials.getAccountName()))
```

Comment:
It would be perfect if this URL came from configuration. This is already done this way for functional tests.

Reply:
I think windows.net won't work even with blob router storage account.

Reply:
yep. need to add app gw